### DISCUSSION TITLE
Setting a default ability to a macro

### DISCUSSION THREAD

**Discussion by zonfire1070** [Hekili/hekili#1886] [Q&A]
I play a BM hunter and I love this addon. As I have been researching to become more efficient when I play it is recommended I make a macro to replace a default ability.

Example: Kill Command  

New Button Maco

#showtooltip Kill Command
/petattack
/use Kill Command

As I expected this does not get the SpellFlash on it. How do I get Hekili to use Spell Flash on my new button in replacement of the old one?

Thank you for any help

**Comment by marekCs** [comment]
Try /cast Kill Command instead of /use or if not helpful try Kill Command spell put in the first row and pet attack at the second

**Reply by zonfire1070** [reply]
That worked. Thank you
